pkg/serverless/appsec/config/config.go

```diff
@@ -70,16 +71,24 @@ func isEnabled() (enabled bool, set bool, err error) {
 	}
 }
 
-func newConfig() (*Config, error) {
+// IsStandalone returns whether appsec is used as a standalone product (without APM tracing) or not
+func IsStandalone() bool {
+	value, set := os.LookupEnv(tracingEnabledEnvVar)
+	enabled, _ := strconv.ParseBool(value)
+	return set && !enabled
+}
```

Comment:
This function call is costly so I suggest you cache the result to avoid extra latencies in the request hot path where you use it.

Reply:
6050d529532bd634